lib/active_admin/batch_actions/views/selection_cells.rb

```diff
@@ -7,8 +7,11 @@ module BatchActions
     class ResourceSelectionToggleCell < ActiveAdmin::Component
       builder_method :resource_selection_toggle_cell
 
-      def build
-        input type: "checkbox", id: "collection_selection_toggle_all", name: "collection_selection_toggle_all", class: "toggle_all"
+      def build(label_text='')
```

Comment:
I want to normalize the style here and use spaces around the equal sign here (https://github.com/activeadmin/activeadmin/pull/4990). If the others agree, could you fix the tiny issue here?